src/NHSD.BuyingCatalogue.Ordering.Api/Startup.cs

```diff
@@ -11,6 +12,8 @@ namespace NHSD.BuyingCatalogue.Ordering.Api
 {
     public sealed class Startup
     {
+        private const string BearerToken = "Bearer";
```

Comment:
I would have thought the JWT package would have a constant for this. Would you mind having a quick look to see if there is? No worries if not.

Reply:
Of course. Will let you know 😄

Reply:
Just pushed the change. JwtBearerDefaults.AuthenticationScheme is "Bearer"